### PR TITLE
Implement data->query, fix #7

### LINKED ISSUE #7: Infer EQL query from data
There is a helper function to do this in other libraries like pathom and malli.

### PR DESCRIPTION
**Pull request by souenzzo**

### CODE REVIEW

File: src/autonormal/core.cljc [hunk 1]
```diff
@@ -367,3 +367,36 @@
...
+    ;; pathom uses  `sequential?` here.
+    (coll? data) (transduce (map data->ast)
+                            (fn
+                              ([] {:type :root :children []})
+                              ([result] result)
+                              ([result el] (eql/merge-asts result el)))
+                            data)))
+
+(defn data->query
+  "Helper function to transform a data into an output shape."
```
Comment (lilactown) [inline src/autonormal/core.cljc:398]:
```suggestion
  "Returns an EQL query that matches the shape of `data` passed to it.
  Useful when you have some data already and want to see what an EQL query that
  returns that data would look like."
```

**Review by lilactown — CHANGES_REQUESTED** [review]
This looks great! I'll ask for a small change to the docstring to be a bit clearer and closer to the writing style of the other docstrings.

**Review by souenzzo — COMMENTED** [review]
done

**Comment by lilactown** [comment]
Wonderful! Thank you

**Pull request merged**